src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -16,9 +16,12 @@
                        (rf/dispatch [:communities/load-category-states id])
                        (rf/dispatch [:dismiss-keyboard])
                        (rf/dispatch [:navigate-to-nav2 :community {:community-id id}]))
-      :on-long-press #(rf/dispatch [:bottom-sheet/show-sheet
-                                    {:content (fn []
-                                                [home.actions/actions community-item])}])}
+      :on-long-press #(rf/dispatch
+                       [:bottom-sheet/show-sheet
+                        {:content  (fn []
+                                     [home.actions/actions community-item])
+                         :selected (fn []
+                                     [quo/communities-membership-list-item nil community-item])}])}
```

Comment:
We should try to create Reagent components that behave well with or without the optional props map because that's a strong convention in Reagent and more generally in React components. That is, the caller can decide to pass props and/or the child/children.

The `nil` here would be unnecessary if you changed `communities-membership-list-item` to have two arities, one receiving the props and another without.

Reply:
@ilmotta Could you give an example

Reply:
Sure @jo-mut, arity is a word used to describe the number of arguments a function/macro takes. Clojure supports multiple arity definitions, which is used extensively in Reagent components to allow components to be rendered with/without props, and without forcing callers to pass a `nil` parameter like you did.

References

- https://clojure.org/guides/learn/functions#_multi_arity_functions.
- https://www.braveclojure.com/do-things/, search for the section `Parameters and Arity`.

Reply:
i get you

Reply:
@ilmotta is the `props` relevant here in the first place with or without the component just functions okay

Reply:
> @ilmotta is the props relevant here in the first place with or without the component just functions okay

I'm not sure I understand the comment. As long as you can call `quo/communities-membership-list-item` without the explicit `nil` for the props, and it works, then we're gold ;)